nucypher/cli/characters/stake.py

```diff
@@ -335,43 +350,29 @@ def detach_worker(click_config,
 
 
 @stake.command()
-@_stake_options
-@click.option('--force', help="Don't ask for confirmation", is_flag=True)
-@click.option('--value', help="Token value of stake", type=click.INT)
-@click.option('--lock-periods', help="Duration of stake in periods.", type=click.INT)
-@nucypher_click_config
-def create(click_config,
-
-           # Stake Options
-           poa, light, registry_filepath, config_file, provider_uri, staking_address, hw_wallet,
-           beneficiary_address, allocation_filepath,
-
-           # Other
-           force, value, lock_periods):
+@group_prealloc_character_options
```

Comment:
I think this group of options should be named something like `@group_staker_options`, since it works for both preallocation and normal stakers.